src/components/Modal/Modal.styled.jsx

```diff
@@ -6,15 +6,15 @@ export const Overlay = styled.div`
   position: fixed;
   top: 0;
   left: 0;
-  background: rgba(51, 119, 255, 0.3);
+  background: rgba(0, 0, 255, 0.17);
   display: flex;
   align-items: center;
   justify-content: center;
 `
 export const ModalContainer = styled.div`
   width: 260px;
   max-width: 343px;
-  height: auto;
+  height: 260px;
```

Comment:
This is the global modal component, so, is used in another views and pages. Please change the styles only in this view
```suggestion
  height: auto;
```